server/api/upload.js

```diff
@@ -0,0 +1,81 @@
+const postSchema = require('../../shared/post-schema');
+const db = require('../lib/db');
+
+module.exports = function upload(req, res, next) {
+  // * Anonymous post check
+  if (req.userId === null) {
+    req.userId = 1;
+  }
```

Comment:
Check it, yo!

```js
req.userId ??= 1;
```

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Logical_nullish_assignment

Reply:
zamn...